wd.html

```diff
@@ -232,6 +232,11 @@
 <pre><code class=rb>require "selenium-webdriver"
 driver = Selenium::WebDriver.for :firefox</code></pre>
 
+<p>If you prefer not to set geckodriver's location using PATH,
+ set the geckodriver binary location using :
+
+<pre><code class=java>System.setProperty("webdriver.gecko.driver", "/path/to/geckodriver");</code></pre>
```

Comment:
You can also set it on the command line using `-Dwebdriver.gecko.driver=/path/to/geckodriver` which is a better way of doing these things since then you don't need to modify source to change the location.